docs/ansible.posix.firewalld_module.rst

```diff
@@ -153,7 +153,7 @@ Parameters
                 </td>
                 <td>
                         <div>Should this configuration be in the running firewalld configuration or persist across reboots.</div>
-                        <div>As of Ansible 2.3, permanent operations can operate on firewalld configs when it is not running (requires firewalld &gt;= 3.0.9).</div>
+                        <div>As of Ansible 2.3, permanent operations can operate on firewalld configs when it is not running (requires firewalld &gt;= 0.3.9).</div>
```

Comment:
It is my understanding that `docs/ansible.posix.firewalld_module.rst` is generated from the docstring of firewalld.py.
So I would request you to change this fix to `firewalld.py` side instead of `ansible.posix.firewalld_module.rst`.

* https://github.com/ansible-collections/ansible.posix/blob/main/plugins/modules/firewalld.py#L84

Also, can you create a fragments file like `227-<TITLE>.yml` here:[fragments](https://github.com/ansible-collections/ansible.posix/tree/main/changelogs/fragments) as `trivial` according to the following docs :

* https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment

Reply:
@saito-hideki Thanks for the answer. I did a new pull request: https://github.com/ansible-collections/ansible.posix/pull/233.